app/models/chargeback_rate_detail.rb

```diff
@@ -37,6 +39,16 @@ def charge(relevant_fields, consumption)
   def metric_value_by(consumption)
     return 1.0 if fixed?
 
+    if quota?
+      quota_items = if consumption.resource.class == ContainerProject
+                      consumption.resource.container_quota_items
+                    elsif consumption.resource.responds_to?(:container_project)
+                      consumption.resource.container_project.container_quota_items
+                    end
+
+      item = quota_items.find_by(:resource => metric)
+      return item.present? ? item.normalize_field_for_chargeback(:quota_enforced) : 0
+    end
```

Comment:
Definitely yes. i was looking for where exactly to put this since i saw a lot of consumption\chargeback-helpers.